packages/common/src/store/ui/own-profile/slice.ts

```diff
@@ -0,0 +1,30 @@
+import type { PayloadAction } from '@reduxjs/toolkit'
+import { createSlice } from '@reduxjs/toolkit'
+
+import { Nullable } from 'utils/index'
+
+export type OwnProfileState = typeof initialState
+
+type State = {
+  trackCount: Nullable<number>
```

Comment:
it's somewhat misleading to call it count as we only count the first page of results, and since we're only using this as a zero vs non-zero value, perhaps we should just store the bool or null